### PR TITLE
Added command completeQuest

### PR DESCRIPTION
**Pull request by mfventura**
new command completeQuest will complete and get reward without going back to giver NPC

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -17,7 +17,7 @@ apply plugin: 'com.github.johnrengelman.shadow'
 apply plugin: 'org.spongepowered.mixin'
 
 group = 'flash'
-version = '1.1.4v2'
+version = '1.1.5v2'
```
Comment (FlashHUN) [inline build.gradle]: Version should be 1.1.5, not 1.1.5v2. Also needs to be modified in the [mods.toml](https://github.com/FlashHUN/FlashNPCs/blob/mfventura/complete-quest-give-reward-command/src/main/resources/META-INF/mods.toml)

File: src/main/java/flash/npcmod/commands/QuestsCommand.java [hunk 2]
```diff
@@ -121,6 +122,45 @@ private int completeObjective(CommandSourceStack source, Player player, String q
     }
     return 0;
   }
+  
+  /**
```
Comment (FlashHUN) [inline src/main/java/flash/npcmod/commands/QuestsCommand.java]: The code itself should explain what it does, not the comments (everybody skips over those anyway... There's a reason why they're similarly colored to the background in most IDEs). Javadocs are not necessary for non-API code, of which there is none atm.

Also, the "completeAllObjectives" parameter is misspelled and you should probably give it a better name.

**Review by FlashHUN — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (mfventura) [reply build.gradle]: Done

File: src/main/java/flash/npcmod/commands/QuestsCommand.java (hunk 2 above)
Comment (mfventura) [reply src/main/java/flash/npcmod/commands/QuestsCommand.java]: Done

**Review by FlashHUN — APPROVED** [review]

**Pull request merged**